LICENSE

```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) 2022 Collab-Community
+Copyright (c) 2022 collab-community/Leetcode-problem
```

Comment:
```suggestion
Copyright (c) 2022 collab-community/Leetcode-problem
```

The copyright is the name of the person, company, organization who owns the copyright, we don't need to inculde the project name haha

Reply:
ok got it, I thought the copyright should be on the name of the project haha

Reply:
That works too, but not required!

Reply:
and not preferred as well.